tests/e2e/e2e_helm_install_test.go

```diff
@@ -0,0 +1,16 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test osm control plane installation with Helm", func() {
+	Context("Using default values", func() {
+		It("installs osm control plane successfully", func() {
+			// Install OSM with Helm
+			Expect(td.HelmInstallOSM()).To(Succeed())
```

Comment:
most of the tests will run a `WaitForPodsReady` function, and wait for the right number of pods expected to be ready in `osmNamespace`; this ensures to some degree that things deployed and are running.

Reply:
oh nvm, I see you expect that from `--wait` correct?

Reply:
I'm using the `--wait` flag on the helm install command which ensures that the Pods installed via the helm chart are in a ready state. I feel like that is a more targeted approach. What do you think? Are there benefits of using `WaitForPodsReady` over the wait flag?

Reply:
I would assume helm knows better which and how many pods does it have to see running? Normally we have to write down the number of pods _we_ expect to see running based on the installation we code, but I'd assume helm does and knows that better. 
No problem with it for me.